File: android/src/main/java/io/adrop/bridge/AdropMethod.kt
package io.adrop.bridge

object AdropMethod {
    const val INITIALIZE = "initialize"
    const val LOAD_BANNER = "loadBanner"
    const val CREATED_BANNER = "createdBanner"

    const val DID_RECEIVE_AD = "onAdReceived"
    const val DID_CLICK_AD = "onAdClicked"
    const val DID_FAIL_TO_RECEIVE_AD = "onAdFailedToReceive"
}

File: ios/bridge/AdropMethod.swift
import Foundation

struct AdropMethod {
    static let INITIALIZE = "initialize"
    static let LOAD_BANNER = "loadBanner"
    static let CREATED_BANNER = "createdBanner"

    static let DID_RECEIVE_AD = "onAdReceived"
    static let DID_CLICK_AD = "onAdClicked"
    static let DID_FAIL_TO_RECEIVE_AD = "onAdFailedToReceive"
}
